app/src/main/java/com/asfoundation/wallet/billing/vkpay/VkPaymentIABViewModel.kt

```diff
@@ -54,8 +54,8 @@ class VkPaymentIABViewModel @Inject constructor(
 
   var transactionUid: String? = null
   var walletAddress: String = ""
-  private val JOB_UPDATE_INTERVAL_MS = 15 * DateUtils.SECOND_IN_MILLIS
-  private val JOB_TIMEOUT_MS = 60 * DateUtils.SECOND_IN_MILLIS
+  private val JOB_UPDATE_INTERVAL_MS = 5 * DateUtils.SECOND_IN_MILLIS
+  private val JOB_TIMEOUT_MS = 600 * DateUtils.SECOND_IN_MILLIS
```

Comment:
10 min looks a lot to give a feedback to user

Reply:
This 10 minutes start as soon as the user is prompted to introduce de payment details. This is a workaround because the success callback from the vk sdk is not called